py/sight/widgets/decision/trials.py

```diff
@@ -288,7 +296,7 @@ def start_jobs(
   if FLAGS.env_name:
     command += f' --env_name={FLAGS.env_name}'
 
-  logging_path = f'gs://{os.environ["PROJECT_ID"]}-sight/d-sub/logs/'
+  logging_path = f'gs://{os.environ["PROJECT_ID"]}-sight/d-sub/meet-logs/{sight.params.label}/{append_ist_time_to_logging_path_12hr()}/'
```

Comment:
Why not use sight.id like above?

Reply:
forgot to remove meet- from the string but the final one is like this
`logging_path = f'gs://{os.environ["PROJECT_ID"]}-sight/d-sub/logs/{sight.params.label}/{append_ist_time_to_logging_path_12hr()}/`

using only sight.id, as this bucket contains 500+ logs, bit difficult to navigate to the location, we thought adding label with date time will be easy to sort the logs via descending order and the user it generated by, let me know your thoughts on this

Reply:
good idea. Yeah, label will help